build.sbt

```diff
@@ -17,17 +21,30 @@ inThisBuild(
 )
 
 lazy val jibCommon = (project in file("jib-common")).settings(
-  name               := "jib-common",
-  crossScalaVersions := List(scalaVersion.value, "2.11.12", "2.13.16", "3.3.6"),
```

Comment:
Do you need `jib-common` to be built against 2.13 too? Did this serve some purpose?